src/mesido/esdl/esdl_heat_model.py

```diff
@@ -1863,9 +1863,14 @@ def convert_gas_demand(self, asset: Asset) -> Tuple[Type[GasDemand], MODIFIERS]:
         ]
         # DO not remove due usage in future
         # hydrogen_specfic_energy = 20.0 / 1.0e6
+        specific_energy = (
+            self.get_internal_energy(asset.name, asset.in_ports[0].carrier) / 10
+        )  # J/g #TODO: is not the HHV for hydrogen, so is off
```

Comment:
HHV = Higher Heating Value
there seemed to be something off in the value for the specific_energy of hydrogen.

Reply:
I still do not understand.I would expect the text to say something like: "This value being used is .... [units], and it is not the Higher Heating Value [Units] yet. The latter value differed from what is expected because of .... . This still has to be resolved. "